<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>275</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblLoading">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>500</width>
      <height>350</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgba(0,0,0,200);color:white;</string>
    </property>
    <property name="text">
     <string>Processing ...</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenLog">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select  Directory</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectXML">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>90</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select  XML</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSelectOutput">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Output Dir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblSelectedDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblXMLDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutputDirectory">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>500</width>
      <height>25</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnStartProcess">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>300</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="copyCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Copy founded files?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="occurTogetherCheck">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>201</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Check failures occur together?</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="DeviceSpecificCheck">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>131</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Is device specific?</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboRegion">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDevice">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Device Number</string>
    </property>
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="placeholderText">
     <string>Device Number</string>
    </property>
   </widget>
   <zorder>btnOpenLog</zorder>
   <zorder>btnSelectXML</zorder>
   <zorder>btnSelectOutput</zorder>
   <zorder>lblSelectedDirectory</zorder>
   <zorder>lblXMLDirectory</zorder>
   <zorder>lblOutputDirectory</zorder>
   <zorder>btnStartProcess</zorder>
   <zorder>copyCheck</zorder>
   <zorder>occurTogetherCheck</zorder>
   <zorder>DeviceSpecificCheck</zorder>
   <zorder>comboRegion</zorder>
   <zorder>txtDevice</zorder>
   <zorder>lblLoading</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
